Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/features/settings/ui/SettingsFragmentPresenter.java

```diff
@@ -806,6 +806,37 @@ private void addEnhanceSettings(ArrayList<SettingsItem> sl)
     sl.add(new SingleChoiceSetting(mContext, IntSetting.GFX_ENHANCE_MAX_ANISOTROPY,
             R.string.anisotropic_filtering, R.string.anisotropic_filtering_description,
             R.array.anisotropicFilteringEntries, R.array.anisotropicFilteringValues));
+    AbstractIntSetting filteringSetting = new AbstractIntSetting()
+    {
+      @Override public int getInt(Settings settings)
+      {
+        return IntSetting.GFX_ENHANCE_FORCE_TEXTURE_FILTERING.getInt(settings);
+      }
+
+      @Override public void setInt(Settings settings, int newValue)
+      {
+        BooleanSetting.GFX_ENHANCE_FORCE_FILTERING.setBoolean(settings, (newValue > 0));
```

Comment:
This setting doesn't exist in the C++ code anymore.

Reply:
Resolved in #11518